### ISSUE TITLE
[AI + Bias] Create Breaking Bias Moodboard + Branding Assets

### ISSUE THREAD

**Issue by sabrinang** [MozillaFoundation/Design#819]
**Context**
Meet with Xavier this afternoon. The first publication of Breaking Bias will be on April 29th. These will be around 500-700 words using the wagtail blog format.

Create a moodboard, gather resources and create branding assets for the Breaking Bias series that goes out every other Thursday. 

Ideas include:
- portrait photography mashup headers and photography treatment
- motion typography for videos and sourcing background audio
- templatized art assets that we can use throughout the series

Format includes:
- blog post assets
     - photo/illustration
     - brainstorm potential components
- social media 
     - instagram story format
- audio/video motion typography clips (blog embed and social)

**Timeline Estimate**
April 20 — moodboard, branding assets
April 22 — finalize text/audio, create article specific assets
April 27 — assets ready for review
~~April 29 — publish blog and socials~~

**Comment by sabrinang** [comment]
Moodboarding and explorations in google slides: https://docs.google.com/presentation/d/1vguyFShTLqaFE7TOWlZknV8EMoWQ-yerpl2Pl3ju5ug/edit?usp=sharing

Explorations and drafts of collage/color with portraiture, abstract shapes and photo cutouts and typography for the series title.
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/2492510/115792489-3d53ac80-a37f-11eb-8667-c954aae22ba6.png">

I shared the .ai file and explorations at crit with Natalie & Nancy to open up if they wanted to jam ideas with me. Also, mentioned to Nat we can coordinate with Xavier if there are video assets in the future.

**Comment by sabrinang** [comment]
Reached out to Xavier that I need the draft content to start creating article specific assets soon and shared with him the slide progress as well.

**Comment by sabrinang** [comment]
Today he mentioned he's still waiting for the second person in the story along with text itself. He said Audrey would want assets for IG grid and IG story that includes pull quotes from the text. I'm blocked by texts but can continue to iterate on the visual direction from feedback.

Xavier forwarded the approved portrait we could use via email on the weekend _(no permissions to use the one I iterated with)_. 

On Friday, I got feedback from Xavier as well:

- Heading: Play with the fact that the letters “AI” are in both words. He thinks the slanted line look looks cool but almost looks too serious for what he had in mind for the series. He liked the middle column of stacked headings.
- Visuals: Good to have lede art for Breaking Bias that doesn’t involve interviewees’ faces in case no one gets back to Xavier and to have free-to-use images as backup or use no people in case it is unbalanced representation
- Xavier still plans to aim to publish on April 29

**Comment by sabrinang** [comment]
Xavier mentioned this morning that they're pushing back the launch of Breaking Bias today. This will also push back the rest of the series plan of Becca's post too until the first post is sorted. The initial post is waiting on another expert for the first article so I'm blocked on content specific photography/portraiture and quotes until further notice. 
_________

I've made progress on visual brand direction earlier this week. The feedback I got from this round of iterations was positive for lightening the mood of the series and incorporating more human qualities with photo collage and torn elements. I explored monospace type so the "ai" letters can be stacked and read vertically in the display heading.  

@kristinashu for review? I iterated on draft assets with the direction that had the best response to refine/explore the headline treatment/capitalization and layout composition with breakout pixel motifs. 
_Slide process:_ https://docs.google.com/presentation/d/1vguyFShTLqaFE7TOWlZknV8EMoWQ-yerpl2Pl3ju5ug/edit#slide=id.gd5b9b31e3f_0_17
_Redpen:_ https://redpen.io/p/np7e200c24b3d23583

I will get feedback from designers at crit this afternoon as well. 

**Note:** in retrospect, I should of divided this ticket up into two (separating moodboard and to establish the series direction explorations from brand assets themselves) as they both took a bit of time and one is blocked and the other is in review.

**Comment by kristinashu** [comment]
Loving all of these and I've left a few comments! But because this is very visual, tied to @natalieworth previous Instagram work, and is closely tied to the Amaya work, it think she would be a better reviewer for this work.

**Comment by natalieworth** [comment]
@sabrinang I left some small comments but overall R+ the darker versions. It's looking really great! I'd say in summary my preference is the darker options since it makes the content feel more serious and will be more eye-catching on the white pages of our site/ email. I also like the textured options—they feels more related to our brand textures and other materials. I also prefer the lowercase type (related more to the moz logo and it feels slightly more friendly).

**Comment by sabrinang** [comment]
Thank you for all the visual feedback Kristina and Natalie!

**I refined a few elements:**
- display heading is all lowercase and cleaned up shadows/alignment
- removed full mozilla wordmark and used 'm' in lower right corner on all instagram post/stories/blog image
- using nunito sans for all quote body copy (made it larger if I could when it was a shorter quote)

**Just an update on the series overall:**
- _Monday:_ Xavier finished interviewing the second person from Color of Change and wants to include their quote along side the first Johanna's article
- _Tuesday:_ Aim for Thursday to release series is back on and I iterated on Johanna's asset but Xavier is undecided if he wants the face shown as it may be too prominent to look like one person's series so I made a type version instagram post in case. Waiting on Johanna's summary or quote content. Collected CoC logo and iterated on instagram assets incorporating their logo colors. Also collecting photography headshots for the next one from folks at the Markup. 
- _Today:_ Waiting for content approval but it looks like we would push back release (not tomorrow) and I'm waiting on the text content still for some instagram panels. I mentioned that since it isn't released yet we should take the time to have a good start as no one is anticipating it yet and gives us a head start when we do execute these biweekly it'll give us some wiggle room. _TBD we may be holding off a bit but will update when I know more._
- I completed Umang's assets (the second in the series)

**Comment by sabrinang** [comment]
_Note:_ I changed the estimate to an 8 as it was more complex than my initial estimate so I wanted to capture there was more content coordination/planning that entailed for this sprint. 

I filed a new issue https://github.com/MozillaFoundation/Design/issues/844 to follow up this work that focuses on series 1 specific assets since the moodboard/branding is complete from this ticket — closing!
